paper/tex/introduction.tex

```diff
@@ -1,63 +1,40 @@
-%We explore a number of performance metrics in this paper, in the controlled environment of the representative training data in order to establish a method for comparing photo-$z$ PDFs derived by different methods.
-%not to make a conclusion regarding the superiority or even relative favorability of each code but to establish a method for comparing photo-$z$ PDFs derived by different methods.
-%These test conditions set the stage for addressing in a future paper the crucial issue of incomplete and non-representative prior information.
-
-
-
-The outline of the paper is as follows: in \S\,\ref{sec:sims} we present the simulated data set; in \S\,\ref{sec:pzcodes} we describe the current generation codes employed in the paper;  in \S\,\ref{sec:metrics} we discuss the interpretation of photo-$z$ PDFs in terms of metrics of accuracy; in \S\,\ref{sec:results} we show our results and compare the performance of the codes; in \S\,\ref{sec:discussion} we offer our conclusions and discuss future extensions of this work.
+The current and next generations of large-scale galaxy surveys, including the Dark Energy Survey \citep[\proj{DES},][]{Abbott:05}, the Kilo-Degree Survey \citep[\proj{KiDS},][]{de_Jong:13}, Hyper Suprime-Cam Survey \citep[\proj{HSC},][]{Aihara:2018a,Aihara:2018b}, Large Synoptic Survey Telescope \citep[\lsst,][]{Abell:09}, Euclid \citep{Laureijs:11}, and Wide-Field Infrared Survey Telescope \citep[\proj{WFIRST},][]{Green:12}, present a paradigm shift from spectroscopic to photometric galaxy catalogues of substantially larger size at a cost of lacking complete spectroscopically confirmed redshifts ($z$).
+
+Effective astrophysical inference using the catalogues resulting from these ongoing and upcoming missions, however, necessitates accurate and precise photometric redshift (\pz) estimation methodologies.
+As an example, in order for \pz\ systematics to not dominate the statistical noise floor of \lsst's main cosmological sample of $\sim 10^{7}$ galaxies, the \lsst\ Science Requirements Document (SRD)\footnote{available at \url{https://docushare.lsstcorp.org/docushare/dsweb/Get/LPM-17}} specifies that individual galaxy \pz s must have root-mean-square error $\sigma_z < 0.02 (1+z)$, $3 \sigma$ catastrophic outlier rate below $10\%$, and bias below $0.003$.
+Specific science cases may have their own requirements on \pz\ performance that exceed those of the survey as a whole.
+In that vein, the \lsst\ Dark Energy Science Collaboration (\lsstdesc) developed a separate SRD \citep{Mandelbaum:2018} that conservatively forecasts the constraining power of five cosmological probes, leading to even more stringent requirements on \pz\ performance, including those defined in terms of tomographically binned subsamples populations rather than individual galaxies.
```

Comment:
However, some component of the LSST SRD goals are motivated by being able to effectively break up/stack individual galaxy p(z) into tomographic bins.  I need to dig back and look into this, as I'm just remembering, but I think one of the early LSST papers (maybe the Hu Zhan paper?) I think tried to estimate requirements on individual object photo-z's needed in order to break them into 10-15 tomographic bins , and I believe that went into the requirements.  If I can find the reference I'll add some text and the reference.  I mean, in some ways trainZ is an example of this: if all the photo-z's had the correct mean but had sigma_z=0.4 then we can get 10 well separated tomographic bins with minimal overlap.

Reply:
Indeed, the SRD assumes both stacking and tomography -- all the requirements, anywhere, are conditioned on current analysis approaches.  I'm not sure it's in-scope to bring this up.